src/patternfly/components/Button/button.scss

```diff
@@ -303,6 +309,8 @@
     border: var(--#{$button}--BorderWidth) solid;
     border-color: var(--#{$button}--BorderColor);
     border-radius: inherit;
+    transition-timing-function: inherit;
+    transition-duration: inherit;
```

Comment:
Just to confirm - since the initial value of `transition-property` is "all", leaving that out here will set that. Was that intended? If not, `transition: inherit;` should do the trick.

Reply:
eh, not really 😄  I'll fix it.